test/xpu/skip_list_win_arc.py

```diff
@@ -0,0 +1,187 @@
+skip_dict = {
+    # SYCL Compiler on Windows removed the following operations when '-cl-poison-unsupported-fp64-kernels' is on
+    # Hence, skip the following windows specific errors
+    "test_ops_xpu.py": (
```

Comment:
Please remove skip lists in `test_ops_xpu.py`. They have been moved into https://github.com/intel/torch-xpu-ops/pull/859